runtime/j9vm/java11vmi.c

```diff
@@ -1812,3 +1812,11 @@ JVM_IsUseContainerSupport(JNIEnv *env)
 
 	return inContainer ? JNI_TRUE : JNI_FALSE;
 }
+
+#if JAVA_SPEC_VERSION >= 18
+JNIEXPORT jboolean JNICALL
+JVM_IsFinalizationEnabled(JNIEnv *env)
```

Comment:
This `JDK18` JVM function addition fits better in `javanextvmi.c` like https://github.com/eclipse-openj9/openj9/blob/f5069c6c95d44eea18a7e172484cafdd2aa33e60/runtime/j9vm/javanextvmi.c#L178-L184

Reply:
Yes, I'll move it.

Reply:
Done.